packages/client/src/components/Button/Button.tsx

```diff
@@ -5,10 +5,17 @@ import { FC } from 'react';
 
 import { ButtonProps } from './typings';
 
-export const Button: FC<ButtonProps> = ({ type = 'button', text, onClick, className, variant }) => {
+export const Button: FC<ButtonProps> = ({ type = 'button', text, onClick, className, variant, testId }) => {
   const buttonClassNames = cn('button', className, { [`button_${variant}`]: variant });
+
+  const props: Record<string, string> = {};
+
+  if (testId) {
+    props['data-testid'] = testId;
+  }
+
   return (
-    <button type={type} onClick={onClick} className={buttonClassNames}>
+    <button type={type} {...props} onClick={onClick} className={buttonClassNames}>
```

Comment:
Раз мы передаем через spread параметры, то можно тогда убрать onClick, т.к. они передаются в <button> без каких-либо изменений
```suggestion
    export const Button: FC<ButtonProps> = ({ type = 'button', text, className, variant, testId }) => {
    ....
    <button {...props} type={type} className={buttonClassNames}>
```